spec/sms_spec.rb

```diff
@@ -0,0 +1,8 @@
+require_relative '../lib/sms.rb'
+
+describe SMS do
+  it 'sends a confirmation text to the user' do
```

Comment:
Useful reading around this kind of test here: https://github.com/makersacademy/takeaway-challenge/blob/main/docs/review.md#tests-should-test-real-behaviours-not-stubs

You want to test the behaviour of the method under test, rather than accidentally stubbing that behaviour, and testing the stub itself. 

To implement a stub for the messenger for your tests, you'll need dependency injection: https://github.com/makersacademy/takeaway-challenge/blob/main/docs/review.md#appropriate-use-of-dependency-injection

I'll share some more resources on this with you on Slack. :)